seed.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Welcome to the seed file! This seed file uses a newer language feature called...
+ *
+ *                  -=-= ASYNC...AWAIT -=-=
+ *
+ * Async-await is a joy to use! Read more about it in the MDN docs:
+ *
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function
+ *
+ * Now that you've got the main idea, check it out in practice below!
+ */
+const db = require('./server/db')
+const {User} = require('./server/db/models')
+
+async function seed () {
+  await db.sync({force: true})
+  // Whoa! Because we `await` the promise that db.sync returns, the next line will not be
+  // executed until that promise resolves!
+
+  const users = await Promise.all([
+    User.create({email: 'cody@email.com', password: '123'}),
+    User.create({email: 'murphy@email.com', password: '123'})
+  ])
+  // Wowzers! We can even `await` on the right-hand side of the assignment operator
+  // and store the result that the promise resolves to in a variable! This is nice!
+  console.log(`Seeded ${users.length} users`)
+  console.log(`Seeded successfully`)
+  process.exit(0)
```

Comment:
I don't like `process.exit` in general. Node docs [pretty much say](https://nodejs.org/api/process.html#process_process_exit_code) "do not actually use this, set an exit code manually and wait for the process to self-terminate if possible."

> It is important to note that calling `process.exit()` will force the process to exit as quickly as possible even if there are still asynchronous operations pending that have not yet completed fully, including I/O operations to `process.stdout` and `process.stderr`.

> In most situations, it is not actually necessary to call `process.exit()` explicitly. The Node.js process will exit on its own if there is no additional work pending in the event loop. The `process.exitCode` property can be set to tell the process which exit code to use when the process exits gracefully.

> …Rather than calling `process.exit()` directly, the code *should* set the `process.exitCode` and allow the process to exit naturally by avoiding scheduling any additional work for the event loop.

You can make the seed file terminate instead of hanging for ~20 seconds by using `db.close()` to drain the connection pool. Annoyingly, `db.close` uses but does not return Bluebird promises, leading to a warning in the console unless you `return null` inside a Bluebird promise chain. Yay Sequelize magic! However, this is made easier from within `async` functions which return native `Promise`s (unless you over-wrote the global `Promise`).

I still think this is much better practice than teaching our students to wantonly wield `process.exit` without fully understanding its implications.